docs/simple_examples.rst

```diff
@@ -0,0 +1,148 @@
+******************************
+Simple scintillometry examples
+******************************
+
+Consider observing a pulsar using a single radio dish.
+Let's assume that the pular is stationary w.r.t. Earth,
+that its radio spectrum is flat,
+... OTHER ASSUMPTIONS ...
+and that the observation is without any noise.
+The pulsar's radiation reaches the Earth via a direct line of sight,
+as shown in the schematic below.
+
+.. plot::
+    :context: reset
+
+    from astropy import units as u
+    import matplotlib.pyplot as plt
+    from example_figure import gen_magnifs, example_figs
+
+    theta = [0.] << u.mas
+    magnifs = gen_magnifs(theta)
+
+    example_figs(theta, magnifs, mu_eff=0.*u.mas/u.yr, velocity=False)
+
+The dynamic spectrum of this observation will look rather plain
+
+A scattering screen
+===================
+
+Now introduce a scattering screen somewhere in the space between the pulsar and
+Earth. In this simple example, the screen will give rise to point-like
+scattering images of the pulsar.
+
+.. plot::
+   :context: reset
+
+    from astropy import units as u
+    import matplotlib.pyplot as plt
+    from example_figure import gen_magnifs, example_figs
+
+    theta = [-2., 0.] << u.mas
+    magnifs = gen_magnifs(theta)
+
+    example_figs(theta, magnifs, mu_eff=0.*u.mas/u.yr, velocity=False)
+
+The effect of relative motion
+=============================
+
+Until now we have assumed there is no relative motion between the pulsar,
+Earth, and the scattering screen (i.e., the interstellar medium).
+While in reality all these components will have their own space motion,
+they can all be included by a single **effective proper motion** with which
+the line of sight moves w.r.t. the scattering screen.
+Including this relative motion (represented in the schematic below by the
+arrow next to the pulsar) will result in 
+
+.. plot::
+   :context: reset
+
+    from astropy import units as u
+    import matplotlib.pyplot as plt
+    from example_figure import gen_magnifs, example_figs
+
+    theta = [-2., 0.] << u.mas
+    magnifs = gen_magnifs(theta)
+
+    example_figs(theta, magnifs)
+
+Two beams
+=========
+
+An single additional scattering image in the lens will result in
+
+.. plot::
+   :context: reset
+
+    from astropy import units as u
```

Comment:
Not very important, but is it possible to re-use the three import lines throughout the file? (maybe by defining a context?)

Reply:
Hmmm, in the Astropy docs example I was following the imports are repeated. We could put some things in the `plot_pre_code` in `conf.py`, but to me that seems like making the code harder to debug.

Reply:
I was hoping we could those common bits in a separate section in the same file. If that's not easily possible, then definitely just keep things as they are.